bitwarden_access_manager.py

```diff
@@ -0,0 +1,715 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Chorus One
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# A copy of the License has been included in the root of the repository.
+
+"""
+Bitwarden Access Manager
+
+Compare the current state of a Bitwarden organization with a desired state
+expressed in a TOML file. Currently this tool only points out the differences,
+it does not automatically reconcile them for you.
+
+USAGE
+
+    ./bitwarden_access_manager.py organization.toml
+
+ENVIRONMENT
+
+Requires BITWARDEN_CLIENT_ID and BITWARDEN_CLIENT_SECRET to be set in the environment.
+Those must contain OAuth2 client credentials for the organization. Only Bitwarden
+members of the organization with OWNER role have access to those credentials.
+
+You can view the credentials at https://vault.bitwarden.com/#/organizations/<organization_id>/settings/account
+
+CONFIGURATION
+
+* The access_all key for members and groups is optional, default is false.
+* The member_access key for a collection only list members with direct access
+to collection. It omits direct access for members with the role
+owners or admins because they have implicit access to all collections.
```

Comment:
It turns out, this is not true, I thought it was because I could access everything in our organization, but it was because `access_all` was true for me (which I discovered using this tool :D). So could we remove this special case?